src/libs/state.js

```diff
@@ -41,8 +41,6 @@ export const knownBucketRequesterPaysStatuses = Utils.atom({})
 
 export const requesterPaysProjectStore = Utils.atom()
 
-export const workspaceBucketLocationStore = Utils.atom()
```

Comment:
By removing this state, it re-introduces the bug you found that I fixed with WorkspaceContainer. See this comment: https://github.com/DataBiosphere/terra-ui/pull/2680#issuecomment-973167744 

To reproduce the bug:

1) Navigate to a workspace with a non-US bucket. Open the ComputeModal. Notice it says `This cloud environment will be created in the region $region.` Click customize. Notice the location dropdown says `$region`.
2) Close the compute modal, and switch to a different tab (say data). Open the Cloud Environment modal here. Notice it now says  `This cloud environment will be created in the region us-central1.` Click customize. Notice the location dropdown says `us-central1`. It shouldn't say us-central1, it should say $region. 

The issue is switching tabs creates a new WorkspaceContainer, but doesn't call refreshWorkspace. Thus, the need for the workspaceBucketLocationStore state.